packages/sanity/src/desk/comments/src/hooks/useCommentOperations.ts

```diff
@@ -52,18 +54,28 @@ export function useCommentOperations(
 
   const authorId = currentUser?.id
 
-  const {documentTitle, toolName, url, workspaceTitle} = useNotificationTarget({
-    documentId,
-    documentType,
-  })
+  const activeToolName = useRouterState(
+    useCallback(
+      (routerState) => (typeof routerState.tool === 'string' ? routerState.tool : undefined),
+      [],
+    ),
+  )
+  const {tools} = useWorkspace()
+  const activeTool = useMemo(
+    () => tools.find((tool) => tool.name === activeToolName),
+    [activeToolName, tools],
+  )
```

Comment:
This was moved out of `useNotificationTarget` since the tool name isn't part of the notification object